application/core/LSYii_Application.php

```diff
@@ -717,4 +717,93 @@ public function getPublicBaseUrl($absolute = false)
         }
         return $baseUrl;
     }
+
+	/**
+	 * Is REST request
+	 *
+	 * @return boolean
+	 */
+    public function isRestRequest()
```

Comment:
Should this rather be a method of the request class? CHttpRequest? Dunno if it's subclassed already.